templates/web/fixamingata/report/new/form_user.html

```diff
@@ -0,0 +1,15 @@
+<h2 class="form-section-heading form-section-heading--private">Andra uppgifter</h2>
```

Comment:
Noting this template is (I believe) included because this string was missed in translation until 2.0.2, so could then be removed with an upgrade.